github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
+============
+
+**Build Badges**
+
+Short project description. You can use the client's pitch. No more than ten lines.
+
+**Table of Contents**
+
+* [Setup](#setup)
+* [Development](#development)
+* [Deployment](#deployment)
+* [More Sections](#more-sections)
+* [Contribution Guidelines](#contribution-guidelines)
+* [Resources](#resources)
+* [Contacts](#contacts)
+
+## Setup
+
+A set of instructions to install the project. It can start with a list of dependencies, like databases, and finish with the instructions. For instance:
+
+First clone the repository to your file system:
+
+```
+git clone git@github.com:subvisual/guides.git
+```
+
+After, run the setup script:
+
+```
+bin/setup
+```
+
+To finish, open the file `.env` and replace `DB_USERNAME` and `DB_PASSWORD` with the correct username and password for the Postgres user.
+
+## Development
+
+A set of instructions to run the project in development. For instance:
+
+To start your development environment run:
+
+```
+bin/server
+```
+
+This will start the node application, Redis, and Gulp.
+ProjectName uses Gulp to watch and compile frontend assets.
+To build the assets only once, not watching for changes, run:
+
+```
+npm run build
+```
+
+## Deployment
+
+Deploys can be manual or automatic. In both cases, write a short explanation about how it works: if the process is manual write the steps necessary to accomplish it; if the process is automatic, link to the page to consult the deployment status.
```

Comment:
Just something like:

```
To deploy the app run `bin/deploy`
```

This is for manual deploys, which we haven't been using that often now. But I think it is still good to have. If there are more complicated settings like deploy to staging and production, it might say something about that as well.